src/App.js

```diff
@@ -5,135 +5,166 @@
+      <Stack.Screen name='InitialScreen' component={InitialScreen} />
+      <Stack.Screen name='NewWordsScreen' component={NewWordsScreen} />
+      <Stack.Screen name='LoadWordsScreen' component={LoadWordsScreen} />
+      <Stack.Screen name='BackupWords' component={BackupWords} />
+      <Stack.Screen name='ChoosePinScreen' component={ChoosePinScreen} />
+    </Stack.Navigator>
+  );
+};
 
-const DashboardStack = createStackNavigator(
-  {
-    Dashboard,
-    MainScreen,
-  },
-  {
-    initialRouteName: 'Dashboard',
-    headerMode: 'none',
-  },
-);
+const DashboardStack = () => {
+  const Stack = createStackNavigator();
+
+  return (
+    <Stack.Navigator
+      initialRouteName='Dashboard'
+      screenOptions={{
+        headerShown: false,
+      }}
+    >
+      <Stack.Screen name='Dashboard' component={Dashboard} />
+      <Stack.Screen name='MainScreen' component={MainScreen} />
+    </Stack.Navigator>
+  );
+};
 
-const SendStack = createStackNavigator(
-  {
-    SendScanQRCode,
-    SendAddressInput,
-    SendAmountInput,
-    SendConfirmScreen,
-  },
-  {
-    initialRouteName: 'SendScanQRCode',
-    headerMode: 'none',
-  },
-);
+const SendStack = () => {
+  const Stack = createStackNavigator();
+
+  return (
+    <Stack.Navigator
+      initialRouteName='SendScanQRCode'
+      screenOptions={{
+        headerShown: false,
+      }}
+    >
+      <Stack.Screen name='SendScanQRCode' component={SendScanQRCode} />
+      <Stack.Screen name='SendAddressInput' component={SendAddressInput} />
+      <Stack.Screen name='SendAmountInput' component={SendAmountInput} />
+      <Stack.Screen name='SendConfirmScreen' component={SendConfirmScreen} />
+    </Stack.Navigator>
+  );
+};
 
-const CreateTokenStack = createStackNavigator(
-  {
-    CreateTokenDepositNotice,
-    CreateTokenName,
-    CreateTokenSymbol,
-    CreateTokenAmount,
-    CreateTokenConfirm,
-    CreateTokenDetail,
-  },
-  {
-    initialRouteName: 'CreateTokenDepositNotice',
-    headerMode: 'none',
-  }
-);
+const CreateTokenStack = () => {
+  const Stack = createStackNavigator();
+
+  return (
+    <Stack.Navigator
+      initialRouteName='CreateTokenDepositNotice'
+      screenOptions={{
+        headerShown: false,
+      }}
+    >
+      <Stack.Screen name='CreateTokenDepositNotice' component={CreateTokenDepositNotice} />
+      <Stack.Screen name='CreateTokenName' component={CreateTokenName} />
+      <Stack.Screen name='CreateTokenSymbol' component={CreateTokenSymbol} />
+      <Stack.Screen name='CreateTokenAmount' component={CreateTokenAmount} />
+      <Stack.Screen name='CreateTokenConfirm' component={CreateTokenConfirm} />
+      <Stack.Screen name='CreateTokenDetail' component={CreateTokenDetail} />
+    </Stack.Navigator>
+  );
+};
 
-const RegisterTokenStack = createStackNavigator(
-  {
-    RegisterToken,
-    RegisterTokenManual,
-  },
-  {
-    initialRouteName: 'RegisterToken',
-    headerMode: 'none',
-  },
-);
+const RegisterTokenStack = () => {
```

Comment:
🚲 I think we should have a document string here like we see in InitStack.

Reply:
Done on [ae88d09](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/286/commits/ae88d09c6cdb386a0025367f7f7a6713205f4e72)